server/fleet/datastore.go

```diff
@@ -188,7 +188,6 @@ type Datastore interface {
 	// "additional" information as this is not typically necessary for the operations performed by the osquery
 	// endpoints.
 	AuthenticateHost(ctx context.Context, nodeKey string) (*Host, error)
-	MarkHostSeen(ctx context.Context, host *Host, t time.Time) error
```

Comment:
Remember to update the datastore mock.

Reply:
Done!, I'll work on a CI check for this soon.